src/main/java/calculator/MultiplyOperation.java

```diff
@@ -0,0 +1,19 @@
+package calculator;
+
+public class MultiplyOperation implements OperationInterface {
+    @Override
+    public int operationPriority() {
+        return 1;
+    }
+
+    @Override
+    public String operationName() {
+        return "*";
+    }
+
+    @Override
+    public int calculation(int beforeNumber, int afterNumber) {
+        int Result = beforeNumber * afterNumber;
```

Comment:
클래스의 시작은 대문자, 변수의 시작은 소문자입니다 🙂

```suggestion
        int result = beforeNumber * afterNumber;
```